pkg/sysprom/initialize.go

```diff
@@ -437,9 +458,16 @@ func (w *InitConfig) createPrometheusRules() error {
 		},
 	}
 
-	_, err := prometheusRulesClient.Create(promObject)
+	var options metav1.GetOptions
+	_, err := prometheusRulesClient.Get("system-prometheus-rules", options)
 	if err != nil {
-		return fmt.Errorf("Failed to create prometheus rule object. Error: %v", err.Error())
+		_, err = prometheusRulesClient.Create(promObject)
+		if err != nil {
```

Comment:
consider using handleError() to commonize these errors.

Reply:
Not able to do it, handleError in appbert would just error out with log.Fatal
Here we need to return, so not possible to do this.

It would look something like this:
if err = handleError(prometheusRulesClient.Create(promObject)); err != nil {
   return
}

where handleError returns the same error it is passed.